components/infobox/wikis/starcraft2/infobox_skill_ability.lua

```diff
@@ -186,31 +186,28 @@ end
 function Ability:getCostDisplay()
 	local race = string.lower(_args.race or '')
 
-	local minerals = tonumber(_args.min or 0) or 0
-	---@diagnostic disable-next-line: cast-local-type
-	minerals = _MINERALS .. '&nbsp;' .. minerals
+	local minerals = _MINERALS .. '&nbsp;' .. (tonumber(_args.min or 0) or 0)
```

Comment:
```suggestion
	local minerals = _MINERALS .. '&nbsp;' .. (tonumber(_args.min) or 0)
```

Reply:
Ability:getCostDisplay
CustomBuilding:_getCostDisplay
Spell:getResearchCost

does things differently for the same thing